pkg/brimstone/brimstone.go

```diff
@@ -470,18 +478,12 @@ func (b Brimstone) CyberArkPAMCPMEventPut(ctx echo.Context) error {
 	return result.Error
 }
 
-// FindAccountID - given hmslhash return list of accountids from the db
-func (b Brimstone) FindAccountID(hmslhash string) ([]string, error) {
+// FindAccount - given hmslhash return list of accounts from the db
```

Comment:
This is incongruent.   "FindAccounts" (plural) would be congruent with the implementation.  Please rename the function to the plural form.